examples/1.0.0/bnpl-openapi.yaml

```diff
@@ -0,0 +1,457 @@
+openapi: 3.0.0
```

Comment:
should all new examples reference 3.1.x? 

if so, the schema object should be updated appropriately with keywords like `unevaulated*` rather than `additional*`.  If you do update the version to 3.1.x, all of the `no siblings with $ref` comments can be ignored, otherwise, they should be updated to valid OAS 3.0.x.